documentation/builders/event-devices.md

```diff
@@ -0,0 +1,134 @@
+# Event devices
+
+## Background
+Event devices are generic input devices that are exposed in `/dev/input`.
+This includes USB peripherals (Keyboards, Controllers, Joysticks or Mouse) as well as potentially bluetooth devices.
+
+A specific usecase for this could be, if a Zero Delay Arcade USB Encoder is used to wire arcade buttons instead of using GPIO pins.
+
+This functionality was previously implemented under the name of [USB buttons](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/components/controls/buttons_usb_encoder/README.md?plain=1).
+
+The devices and their button mappings need to be mapped in the configuration file.
+
+## Configuration
+
+To configure event devices, first add the plugin as the last entry to the module list:
+
+``` yaml
+  modules:
+    named:
+      event_devices: controls.event_devices # Add as last entry
+```
+
+Then add the following section to the configuration:
+
+``` yaml
+  event_devices:
+    devices: # list of devices to listen for
+      {device nickname}: # config for a specific device
+        device_name: {device_name} # name of the evdev
+        exact: False/True # optional to require exact match
+        mapping:
+          {key-code}: # evdev event id
+              {rpc_command_definition} # eg `alias: toggle`
+```
+The `{device nickname}` is only for your own orientation and can be choosen freely.
+For each device you need to figure out the `{device_name}` and the `{event_id}` corresponding to key strokes, as indicated in the sections below.
+
+### Identifying the `{device_name}`
+
+The `{device_name}` can be identified using the following Python snippet:
+
+``` Python
```

Comment:
In the spirit of run_*.py files, would it make sense to provide this code as a configurator? Or at least as a runable script so users don't have to write the code themselves?